doc/doxygen/headers/glossary.h

```diff
@@ -377,7 +377,32 @@
  * parallel::distributed::Triangulation .
  * </dd>
  *
- * @see @ref boundary "The module on boundaries"
+ * @see @ref boundary "The module on boundaries".
+ *
+ *
+ * <dt class="glossary">@anchor GlossCoarseMesh <b>Coarse mesh</b></dt>
+ * <dd>
+ *   A "coarse mesh" in deal.II is a triangulation object that consists only
+ *   of cells that are not refined, i.e., a mesh in which no cell is a child
+ *   of another cell. This is generally how triangulations are first
+ *   constructed in deal.II, for example using (most of) the functions in
+ *   namespace GridGenerator, the functions in class GridIn, or directly
+ *   using the function Triangulation::create_triangulation(). One can of
+ *   course do computations on such meshes, but most of the time (see, for
+ *   example, almost any of the tutorial programs) one first refines the
+ *   coarse mesh either globally (using Triangulation::refine_global()),
+ *   or adaptively using a refinement criterion, the functions in
+ *   namespace GridRefinement, and then calling
```

Comment:
I don't understand the part `[...] using a refinement criterion, the functions in namespace GridRefinement,[...]` syntactically.
Do you want to say something like `[...] using a refinement criterion (e.g, one of the ones implemented in namespace GridRefinement) [...]`?
Also, adaptive mesh refinement and global refinement are not mutually exclusive, so I would just skip "either".